src/components/candidate-form.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+
+import styled from "styled-components";
+import axios from "axios";
+
+export default function CandidateForm(props) {
```

Comment:
```suggestion
export default function CandidateCard(props) {
```
Form은 input으로 입력받은 정보를 제출하는 section을 의미합니다.
이 component의 역할과는 맞지 않는 것 같아요!